source/content/symlinks-assumed-write-access.md

```diff
@@ -1,15 +1,26 @@
 ---
-title: Using Extensions That Assume Write Access
+title: Symlinks and Assumed Write Access
 description: Learn how to create symbolic links from the code directory to a file.
 tags: [debugfiles]
 categories: [troubleshoot]
+reviewed: "2020-03-13"
 ---
 
-Some modules and plugins create files within hard-coded paths outside of the standard path for the given framework, which can be problematic on Pantheon. WordPress stores files within `wp-content/uploads` and Drupal uses `/sites/default/files`. These directories are symbolically linked to Pantheon's cloud-based filesystem, Valhalla, which is writeable on all environments.
+Some modules and plugins create files within hard-coded paths outside of the standard path for the given framework, which can be problematic on Pantheon.
+
+## Standard File Path
+
+Pantheon provides one location for files that are part of your sites content, like those that are managed through Drupal upload forms, e.g. user profile pictures: `/sites/default/files`. For Drupal sites, this is the only location you can use for files that are uploaded as part of your application.
```

Comment:
You can move the location if you wanted, but as I mentioned above, this is a multi-step process involving symlinks and re-protecting the private directory.

Reply:
Yea, that's what this is saying... this is the only location you can use for files, _by default_. The rest of the doc goes on to explain symlinking.

Reply:
I think https://github.com/pantheon-systems/documentation/pull/5575/commits/440f8ca0cc7227945102e6ea9765f503596366d1 Resolves this.